src/compiler/crystal/semantic/top_level_visitor.cr

```diff
@@ -1027,6 +1027,9 @@ class Crystal::TopLevelVisitor < Crystal::SemanticVisitor
     stripped_doc = node.doc.try &.strip
     if stripped_doc == ":ditto:"
       node.doc = @last_doc
+      # TODO: remove after 0.29.0
```

Comment:
I'd suggest to put the TODO just above the `raise`, when we remove it it'll be obvious that wee don't need the `if` anymore